docs/examples/langchain_lcel_based_agents.py

```diff
@@ -0,0 +1,310 @@
+"""
+Find LCEL-Based Agents that Send Non-Dict Inputs
+
+Based on investigation: LangGraph's create_react_agent uses RunnableSequence
+and sends list inputs to on_chain_start. LangChain has similar LCEL-based agents:
+- create_openai_functions_agent
+- create_tool_calling_agent
+- create_react_agent (LangChain version, not LangGraph)
+
+These likely exhibit the same behavior.
+"""
+
+import os
+
+from dotenv import load_dotenv
```

Comment:
Why do we need this here? if it's test can we move this to appropriate location or convert this to docs. Right now it looks to be something in between.